rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java

```diff
@@ -457,8 +457,11 @@ private void saveReceipts(Block block, BlockResult result) {
     }
 
     private void processBest(final Block block) {
+        logger.debug("Starting to run transactionPool.processBest(block)");
```

Comment:
Could we add the block number and short hash to the log message? It could help when grep over the log lines. I'm not sure about the current format (there is a pull request that standardize the format)

Reply:
Added it directly as MDC keys to the whole block processing